integrations/langfuse/pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai>=2.1.0", "langfuse"]
+dependencies = ["haystack-ai>=2.1.0", "langfuse", "anthropic-haystack", "cohere-haystack"]
```

Comment:
Watch out Anthropic and Cohere are becoming the most popular LLM integrations :-) 
Jokes aside, I don't think there is another way to automatically check proper creation of GENERATION span (see attached image in the PR text) for these LLMs other than `test_tracing_integration` integration method in `integrations/langfuse/tests/test_tracing.py`

Reply:
Why can't these two integrations be lazy-imported?

Reply:
@shadeMe apologies - I moved these dependencies to appropriate place - they are needed for test only.